### ISSUE TITLE
iri90.f,mzgrid.f90, nrlmsise00.f etc

### ISSUE THREAD

**Issue by jiaozilong** [NCAR/GLOW#4]
when compiling using intel fortran compiler, the error is given:
error #7983: The storage extent of the dummy argument exceeds that of the actual argument.   [AP]

**Comment by scivision** [comment]
This fork has been updated to handle these and other problems:

https://github.com/gemini3d/glow

**Comment by jiaozilong** [comment]
thanks a lot! let me try again.

**Comment by jiaozilong** [comment]
thank you! it works

**Comment by caotianyu123456** [comment]
I am wondering the available versions of netcdf cause i couldn't find the available one
